test/manager.js

```diff
@@ -3,7 +3,6 @@
 var _ = require('underscore');
 var async = require('async');
 var session = require('express-session');
-var mysql = require('mysql');
```

Comment:
This removal was carried over from #91 . I didn't know if this require was needed here either?